src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataCollectionSerializer.cs

```diff
@@ -161,17 +174,25 @@ public ODataCollectionSerializer(ODataSerializerProvider serializerProvider)
                     IEdmTypeReference actualType = writeContext.GetEdmType(item, item.GetType());
                     Contract.Assert(actualType != null);
 
-                    itemSerializer = itemSerializer ?? SerializerProvider.GetEdmTypeSerializer(actualType);
-                    if (itemSerializer == null)
+                    if (isForAnnotations && actualType.IsComplex())
                     {
-                        throw new SerializationException(
-                            Error.Format(SRResources.TypeCannotBeSerialized, actualType.FullName()));
+                        valueCollection.Add(
+                      new ODataComplexSerializer(SerializerProvider).CreateODataValue(item, actualType, writeContext).GetInnerValue());
```

Comment:
nit: indent this line under the previous line for readability

Reply:
#Resolved

Reply:
does not look like its addressed in iteration 26.

---
In reply to: [458454146](https://github.com/OData/WebApi/pull/2219#discussion_r458454146) [](ancestors = 458454146)